tracer/src/Datadog.Trace/Activity/ActivityListenerHandler.cs

```diff
@@ -48,16 +48,34 @@ public static bool OnShouldListenTo<T>(T source)
             {
                 if (source is IActivitySource activitySource)
                 {
-                    if (handler.ShouldListenTo(sName, activitySource.Version) && HandlerBySource.TryAdd(sName, handler))
+                    if (handler.ShouldListenTo(sName, activitySource.Version))
                     {
-                        Log.Debug("ActivityListenerHandler: {SourceName} will be handled by {Handler}.", sName, handler);
-                        return true;
+                        if (handler is DisableActivityHandler)
+                        {
+                            Log.Information("ActivityListenerHandler: {SourceName} will not be listened to by the .NET Tracer.", sName);
```

Comment:
Consider using the active voice:
```suggestion
                            Log.Information("ActivityListenerHandler: The .NET Tracer will not listen to {SourceName}.", sName);
```

Reply:
Thanks! That flows a lot better

I updated both spots